src/Network/Network/help/Set-AzApplicationGatewayAutoscaleConfiguration.md

```diff
@@ -33,6 +33,15 @@ The first command gets the application gateway and stores it in $gw variable.
 The second command updates the autoscale configuration from the application gateway.
 The third command updates the application gateway on Azure.
 
+### Example 2
+
+Updates Autoscale Configuration of an application gateway. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+Set-AzApplicationGatewayAutoscaleConfiguration -ApplicationGateway <PSApplicationGateway> -MaxCapacity 4 -MinCapacity 5
```

Comment:
This doesn't seem valid. How can the minimum capacity be more than the maximum capacity?